### ISSUE TITLE
comments are sent as HTML not markdown

### ISSUE THREAD

**Issue by karlcow** [webcompat/webcompat.com#2793]
I have the feeling that webcompat.com is not sending anymore markdown raw text to github but HTML.

It has plenty of consequences on escaping. See comments for example.
https://github.com/webcompat/web-bugs/issues/24472#issuecomment-460929940

I have typed markdown but this is what GitHub has stored.

```
<p>when I first access the page all the buttons have the same color.</p>
<p><a href="https://webcompat.com/uploads/2019/2/675cb7a1-ef72-4d02-ad56-be195940d0e4.jpg" rel="nofollow"><img src="https://webcompat.com/uploads/2019/2/675cb7a1-ef72-4d02-ad56-be195940d0e4-thumb.jpg" alt="Screenshot Description"></a></p>
<p>for me hovering change the color of the button BUT moving away the color doesn't stick. So the issue is not really about the reload but more about keeping the color. (i do not have a touch screen.)</p>
<p>Let me simulate a touch screen. Still the same thing.</p>
<p>so on tap or click, in chrome, the button seems to retain the focus while firefox doesn't</p>
<p>btw, doing</p>
<pre><code class="language-css">      .btn {
        background-color: #4285f4;
      }

      .btn:hover {
        background-color: red;
      }

      .btn:focus {
        background-color: pink;

        /* The outline parameter suppresses the border
        color / outline when focused */
      }

      .btn:active {
        background-color: green;
      }
</code></pre>
<p>makes it more visible.</p>
<p>Firefox keeps the focus only with tab navigation on the button.</p>
<p>Something here is doing testing in between platform.<br>
<a href="https://gist.github.com/cvrebert/68659d0333a578d75372" rel="nofollow">https://gist.github.com/cvrebert/68659d0333a578d75372</a></p>
<p>There are differences it seems in between Firefox on Windows and Firefox on macOS</p>
<p>I'm pretty sure there must be a bug on mozilla bugzilla for this.</p>
<p><a href="https://github.com/masayuki-nakano" rel="nofollow">@masayuki-nakano</a> might know which one.</p>
```

@marimeireles do we convert to HTML the comment before sending it? If yes we need to not do that.

**Comment by marimeireles** [comment]
I'm confused @karlcow.

I think the code responsible for converting from markdown to HTML is this:
https://github.com/webcompat/webcompat.com/blob/d2edc8ce4f303055e3ef7c82e64d730a4c4ca57e/webcompat/static/js/lib/issues.js#L527

Reading `markdown-it` [docs](https://markdown-it.github.io/markdown-it/#MarkdownIt.render) I found out that `md.render`:
>Render markdown string into html. It does all magic for you :).

I think we've always done that way? I'm not sure though!

Of course that if it's a problem I can fix it. But I didn't understand what's wrong with your comment.  Can you please point me what's wrong?

Thanks :)
& sorry for taking so long to answer this issue.

**Comment by karlcow** [comment]
> I think we've always done that way? I'm not sure though!

Nope. We always sent the comment in markdown to GitHub. 
To show what is happening.

Here I created an issue and I just commented with markdown.

<img width="631" alt="capture d ecran 2019-02-13 a 07 33 15" src="https://user-images.githubusercontent.com/505230/52672653-d138bb00-2f61-11e9-8396-fb5c9444e484.png">

markdown-it renders it **locally** as html. NO ISSUE.

But this comment has also been sent to GitHub as html and not markdown. Let's do a reload. 

<img width="637" alt="capture d ecran 2019-02-13 a 07 35 42" src="https://user-images.githubusercontent.com/505230/52672723-fd543c00-2f61-11e9-88c3-cffaba1d77da.png">

Interesting why the colors are gone… (building suspense here 😛 )
Let's go to the test repo.

OOOOOH (dramatic effect 🎭 ) the comment has been converted to HTML and stored as HTML.
https://github.com/webcompat/webcompat-tests/issues/1759

<img width="792" alt="capture d ecran 2019-02-13 a 07 37 00" src="https://user-images.githubusercontent.com/505230/52672817-40aeaa80-2f62-11e9-915c-4d63e7d1db8b.png">

We need to send in the post to GitHub the markdown text. Not the HTML. :)
